src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -854,6 +854,12 @@
           long-summary: |
               Used to control the mode the network plugin should operate in. For example, "overlay" used with
               --network-plugin=azure will use an overlay network (non-VNET IPs) for pods in the cluster.
+        - name: --network-policy
+          type: string
+          short-summary: Update the mode of a network policy.
+          long-summary: |
+              Specify "azure" for Azure network policy manager, "cilium" for Azure CNI Overlay powered by Cilium.
```

Comment:
So there's no way to update the network policy of an existing cluster to calico? And if it's possible to update the network policy to azure, could you please help add a new live test case?

Reply:
If my understanding is correct, in update command, when user specify option value as `""`, the network policy would be disabled (and in rp response this is `none`), please also add a live test case for this scenario.

Reply:
unfortunately, sending RP networkPolicy="" gets ignored in the update due to partial PUT semantics (AKS RP backfills with the current value in the cluster). 20230802preview API adds a "none" value so cx can explicitly disable the network policy.

Reply:
and yes, we will add a live test once we fully support setting networkPolicy="none". RP changes aren't ready yet though.